pkg/server/server_test.go

```diff
@@ -420,7 +420,7 @@ func testServerUseDatabase(ctx context.Context, s *ImmuServer, t *testing.T) {
 	}
 	m := make(map[string]string)
 	m["Authorization"] = "Bearer " + string(dbs.Token)
-	ctx = metadata.NewIncomingContext(ctx, metadata.New(m))
+	metadata.NewIncomingContext(ctx, metadata.New(m))
```

Comment:
actually, last three lines (421-423) may be removed as they are not doing any meaningful validation

Reply:
Removed.